src/shared/components/experiment.tsx

```diff
@@ -60,21 +61,28 @@ export const Experiment: React.FC<IProps> = ({ experiment, data, onDataChange })
 
   return (
     <div className={css.experiment}>
-      <div className={css.sectionButtons}>
-        {
-          sections.map(s =>
-            <SectionButton key={s.title} title={s.title} onClick={setSection.bind(null, s)}/>
-          )
-        }
+      <div className={css.header}>
+        <div className={css.headerBackIcon} onClick={setExperiment?.bind(null, undefined)}>←</div>
+        <div className={css.headerInitialsIcon}>{initials}</div>
+        <div className={css.headerTitle}>TBD</div>
       </div>
```

Comment:
I feel like we might to keep this part of the functionality in a parent component:
1. Maybe it's opinionated, but it doesn't seem to fit here. It feels a bit awkward that `Experiment` component accepts `setExperiment` just to call it with `undefined`. 
2. Probably more important - I think we might want to have this funcionality only in the mobile app. I don't think that LARA should let you go back to experiment picker and set a different experiment. So, I think this could be part of the mobile app components.